src/server.py

```diff
@@ -11,7 +11,8 @@
 import MySQLdb
 from flask import Flask, request, json, Response, abort
 
-static_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "static"))
+SCRIPT_DIR = os.path.abspath(os.path.realpath(os.path.dirname(__file__)))
+static_path = os.path.abspath(os.path.join(SCRIPT_DIR, "..", "static"))
```

Comment:
This can be just:
```
SCRIPT_DIR = os.path.abspath(os.path.dirname(__file__))
static_path = os.path.join(os.path.dirname(SCRIPT_DIR), "static")
```